Autopilot/TelemetryManager/Telemetry_Manager.c

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file Telemetry_Manager.c
+ * @author Jingting Liu
```

Comment:
You need to be able to guarantee stateflow works the way you want. We do this with unit tests. Take a look at how it's done for the attitude manager state machine here:
Autopilot\Test\Src\AttitudeManager\Test_Fsm.cpp

I also have a tutorial for how to set up unit tests here: https://uwarg-docs.atlassian.net/wiki/spaces/TUT/pages/1058078728/test+driven+development

Reply:
Woah thanks for the super detailed comments, I'll fix them accordingly!